inbox/events.xml

```diff
@@ -0,0 +1,564 @@
+    <jid>goffi@jabber.fr</jid>
+  </author>
+  <revision>
+    <version>0.0.1</version>
+    <date>2022-09-08</date>
+    <initials>jp</initials>
+    <remark><p>First draft.</p></remark>
+  </revision>
+</header>
+<section1 topic='Introduction' anchor='intro'>
+    <p>Nowadays, it is common to handle all kind of events through desktop computer or phone: it is useful to have them at hand, to easily share them and to have all kind of reminders.</p>
+    <p>XMPP is a good fit to handle events: it's a communication protocol, and as such it would be useful for it to have tools to handle and share events.</p>
+    <p>The only existing attempt at this point is &xep0097; which describe a way to wrap iCal<note>iCalendar <link url='https://datatracker.ietf.org/doc/html/rfc5545'>https://datatracker.ietf.org/doc/html/rfc5545</link></note> data. However, this XEP never got traction, is not taking advantage of XMPP features, doesn't handle thing such as invitees and RSVP, and it not easily extensible.</p>
+    <p>This XEP propose an alternative which take profit of existing features such as &xep0060;.</p>
+</section1>
+<section1 topic='Requirements' anchor='reqs'>
+  <p>The design goals of this XEP are:</p>
+  <ul>
+      <li>use pubsub and its access model to be able to have events either private or public, and to take profit of the subscription mechanism</li>
+      <li>being able to specify easily simple events with minimum metadata (e.g. doctor appointment, coffee with a friend)</li>
+      <li>provide optional tools such as RSVP or invitees list for events organization</li>
+      <li>it should be possible to optionally ask any kind of extra data in RSVP (e.g.: number of people attending, allergy information for dinner, etc.)</li>
+      <li>group several events of same kind in a pubsub node (e.g. personal agenda, work agenda, events linked to an organization)</li>
+      <li>flexible enough to specify all kind of extra metadata (e.g. description, classification tags, accessibility information) and media (e.g. main picture)</li>
+      <li>being able to optionally link to other pubsub features such as &xep0277; blog, or a way to share picture after the event is finished</li>
+      <li>be extensible</li>
+  </ul>
+</section1>
+<section1 topic='Glossary' anchor='glossary'>
+    <ul>
+        <li><strong>RSVP</strong>: "<link url='https://en.wikipedia.org/wiki/RSVP'>Répondez s'il vous plaît</link>", a way for event invitees to indicate if they plan to participate to the event</li>
+    </ul>
+</section1>
+<section1 topic='Use Cases' anchor='usecases'>
+    <section2 topic='Creating a Simple Event' anchor='simple-event'>
+        <p>Juliet and her nurse decide to have a coffee together. Juliet does know the usual place where they're going, and thus doesn't need to add much informations, she just create an event for her agenda on her XMPP client. The client creates an item on Juliet's PEP node `urn:xmpp:events:0` as follow:</p>
```

Comment:
`does know` should be `knows`
`informations` should be `information`, it's never written with an `s` in English
`she just create an event` should be `she creates an event`